src/main/java/com/snowflake/kafka/connector/internal/streaming/BufferedTopicPartitionChannel.java

```diff
@@ -958,11 +962,16 @@ private long getRecoveredOffsetFromSnowflake(
    *     snowflake.
    */
   private long fetchLatestCommittedOffsetFromSnowflake() {
+    SnowflakeStreamingIngestChannel channelToGetOffset = this.channel;
+    return fetchLatestOffsetFromChannel(channelToGetOffset);
+  }
+
+  private long fetchLatestOffsetFromChannel(SnowflakeStreamingIngestChannel channelToGetOffset) {
```

Comment:
channelToGetOffset -> rename to just the "channel" ?

Reply:
👍